operators/orsi/orsi_segmentation_postprocessor/CMakeLists.txt

```
@@ -28,7 +28,7 @@ add_library(${PROJECT_NAME}
   segmentation_postprocessor.cu
 )
 
-set_target_properties(${PROJECT_NAME}  PROPERTIES CUDA_ARCHITECTURES "70;80")
+set_target_properties(${PROJECT_NAME}  PROPERTIES CUDA_ARCHITECTURES "80")
```

Looks like CUDA Toolkit 13.0 should still support >= 7.5, any concerns about keeping that min version (Quadro RTX compatibility) for now?

```suggestion
set_target_properties(${PROJECT_NAME}  PROPERTIES CUDA_ARCHITECTURES "75;80")
```